web/local.mk

```diff
@@ -1,20 +1,22 @@
-WEB_BABELGEN:=$(patsubst %.babel,%.js,$(shell find web/gen -name '*.babel'))
-WEB_LESSGEN:=$(patsubst %.less,%.css,$(shell find web/gen -name '*.less'))
-WEB_GEN:=$(WEB_LESSGEN) $(WEB_BABELGEN)
-WEB_STATIC:=web/index.html web/res/*.css web/res/*.js
+WEB_SRC:=$(shell find web/src -type f)
+WEB_BUILD:=$(shell find web/build -type f)
```

Comment:
This line (and using the variable as a dependency later) doesn't really make any sense -- you're looking for files that will be generated, so won't necessarily exist at the time this is evaluated.